site/content/docs/cli/npm-packages.md

```diff
@@ -26,4 +29,58 @@ The latest runtime is `2022.10` at the time of writing. This runtime contains am
 - [Browse the latest runtime specs](https://www.checklyhq.com/docs/runtimes/specs/)
 - [Learn more about runtimes](https://www.checklyhq.com/docs/runtimes/)
 
+## Local Dependencies
+
+Browser Checks are also able to import other JavaScript and TypeScript files as dependencies.
```

Comment:
@clample we are using "Browser check" with only the first word capitalized elsewhere in the page and the docs. Do we want to keep this consistent?

Reply:
I do see this also appears as "Browser Checks" in other places. Not sure if the guidelines have changed...

Reply:
@ragog I can update the page to follow best practices. It should be "Browser Checks" everywhere, right?

Reply:
@clample it turns out it is "browser checks" (unless in a headline or at the beginning of a phrase ofc) - see the new Checkly Style Guide on notion.